client/web/compose/src/components/ModuleFields/Editor/File.vue

```diff
@@ -49,6 +64,62 @@
       mode="list"
       class="mt-2"
     />
+
+    <b-modal
+      ref="modal"
+      size="lg"
+      modal-class="video-modal-width"
+      :title="$t('editor.file.webcam.title')"
+      ok-only
+      ok-title="Close"
+      body-class="d-flex flex-column align-items-center"
+      @ok="closeCamera"
+      @show="initializeWebcam"
+    >
+      <video
+        ref="video"
+        width="640"
+        height="480"
+        autoplay
+        style="display:none;"
```

Comment:
Why use display: none, when you can use if/else?
This goes for all of them in this file. Unless there is a specific reason.